openstack/orchestration/v1/stacks/doc.go

```diff
@@ -5,4 +5,121 @@
 // application framework or component specified (in the template). A stack is a
 // running instance of a template. The result of creating a stack is a deployment
 // of the application framework or component.
+
+/* Update Stack examples using the different update methods.
+Summary of behavior between the different methods :
+
+Function | Test Case | Result
+
+Update()	| Template AND Parameters WITH Conflict | Parameter takes priority, parameters are set in raw_template.environment overlay
+Update()	| Template ONLY | Template updates, raw_template.environment overlay is removed
+Update()	| Parameters ONLY | No update, template is required
+
+UpdatePatch() 	| Template AND Parameters WITH Conflict | Parameter takes priority, parameters are set in raw_template.environment overlay
+UpdatePatch() 	| Template ONLY | Template updates, but raw_template.environment overlay is not removed, existing parameter values will remain
+UpdatePatch() 	| Parameters ONLY | Parameters (raw_template.environment) is updated, excluded values are unchanged
+
+The PUT Update() function will remove parameters from the raw_template.environment overlay
+if they are excluded from the operation, whereas PATCH Update() will never be destructive to the
+raw_template.environment overlay.  It is not possible to expose the raw_template values with a
+patch update once they have been added to the environment overlay with the PATCH verb, but
+newly added values that do not have a corresponding key in the overlay will display the
+raw_template value.
+*/
+
+// example stack update using the Update() (PUT) method
+
+/*
+t := make(map[string]interface{})
+f, err := ioutil.ReadFile("template.yaml")
+if err != nil {
+	panic(err)
+}
+err = yaml.Unmarshal(f, t)
+if err != nil {
+	panic(err)
+}
+
+template := stacks.Template{}
+template.TE = stacks.TE{
+	Bin: f,
+}
+
+var params = make(map[string]interface{})
+params["number_of_nodes"] = 2
+
+stackName := "my_stack"
+stackId := "d68cc349-ccc5-4b44-a17d-07f068c01e5a"
+
+stackOpts := &stacks.UpdateOpts{
+	Parameters: params,
+	TemplateOpts: &template,
+}
+
+// client is an instance of *gophercloud.ServiceClient
```

Comment:
Instead of this line, just call client `orchestrationClient` or something like that.